main.py

```diff
@@ -0,0 +1,23 @@
+class Hotel():
+    static = 0
+
+    def __init__(self, name, visitor, number):
```

Comment:
Це формально не впливає на прийняття завдання, але слід одразу звикати давати змінним змістовні назви. Наприклад:

```suggestion
    def __init__(self, name, visitors_per_year, number_of_rooms):
```